ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/landing/LandingScreen.kt

```diff
@@ -93,6 +94,19 @@ fun LandingScreen(
                     )
                 }
             )
+
+            Spacer(modifier = Modifier.size(4.dp))
```

Comment:
Let's remove spacers and replace line 70 (`verticalArrangement = Arrangement.Center`) with

`verticalArrangement = Arrangement.spacedBy(4.dp, Alignment.CenterVertically)`